README.md

```diff
@@ -103,6 +103,13 @@ Some reviews will require 2 👍, based on the characteristics of the PR. Some o
 
 When a second reviewer is added, it's expected that this reviewer will be a member of a senior engineering group. The sr-eng review may add other reviewers as per their discretion.
 
+### Request and response expectations
+
+- Where will we make PR requests?
+- What is the expected response times?
+- How do we escalate requests?
```

Comment:
@efixler @bassrock Any thoughts on this section?  I think we should answer these questions before socializing.

Reply:
@collectedmind Yea, I kinda like the way platform and generation do the slack workflows and a specific channels. Wdyt of that?

Reply:
Some of this has been answered in parallel PRs, see https://github.com/MozillaSocial/pr-template/pull/3 for the latest iteration.

As far as the automated bits go, I'm generally in agreement with what you all propose about them. This document can be most helpful if it gets us aligned on values and expectations, so that then we can get people working to implement these various bits in the new org. 

I think it'd be helpful to keep the _implementation_ parts a little separate and clearly marked from the values and policy, so that we have a todo list that hopefully people will want to collaborate and help on.

Reply:
Maybe it's helpful to merge efixler/rfc-pr into this branch - I think it's potentially a good idea to break out the Request/Response expectations from the Etiquette section, but this'll likely break on a merge.